clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java

```diff
@@ -341,10 +353,7 @@ public static EnumSet<ApiKeys> controllerApis() {
     }
 
     public static EnumSet<ApiKeys> clientApis() {
-        List<ApiKeys> apis = Arrays.stream(ApiKeys.values())
-            .filter(apiKey -> apiKey.inScope(ApiMessageType.ListenerType.BROKER))
-            .collect(Collectors.toList());
-        return EnumSet.copyOf(apis);
+        return brokerApis();
```

Comment:
This simplification is unrelated to the main change in this PR - just something I noticed could be cleaned up.